README.md

```diff
@@ -80,102 +80,113 @@ Edit, enjoy live updates.
+   ```bash
+   sphinx-autobuild -H 127.0.0.1 -b html ./docs/source/ ./docs/build/html
+   ```
 6. Connect to http://localhost:8000 Edit files. Watch live updates. Enjoy.
 
 ## Sphinx Tricks
 
-* If the whole section belongs in the Enterprise Edition, put the following note:
-    ```
-    .. note::
-
-       Role Based Access Control (RBAC) is only available in StackStorm Enterprise Edition. For
-       information about enterprise edition and differences between community and enterprise edition,
-       please see `stackstorm.com/product <https://stackstorm.com/product/#enterprise>`_.
-    ```
-    Refer to Enterprise edition in passing with
-
-        `see Enterprise Edition <https://stackstorm.com/product/#enterprise>`_
+* TODO (Use [http://localhost:8000/todo.html](http://localhost:8000/todo.html) for full TODO list (must be empty when we ship):
 
-* TODO (Use [http://localhost:8000/todo.html](http://localhost:8000/todo.html) for full TODO list (must be empty when we ship)
-:
-
-    .. todo:: Here is a TODO
+  ```rst
+  .. todo:: Here is a TODO
+  ```
 
 * Code fragment:
 
-    .. code-block: bash
+  ```rst
+  .. code-block:: bash
 
       # List all available triggers
-        st2 trigger list
+      st2 trigger list
+  ```
 
 * Reference the document
-```
-       :doc:`/start`
-       :doc:`in the Rules doc </rules>`
-```
-* Referencing an arbitrary section: for instance, there's examples section in sensors.rst. Define a reference on `examples` section in sensors.rst:
 
-         .. _sensors-examples:
+  ```rst
+  :doc:`/start`
+  :doc:`in the Rules doc </rules>`
+  ```
+* Referencing an arbitrary section: for instance, there's examples section in `sensors.rst`. Define a reference on `examples` section in `sensors.rst`:
+
+  ```rst
+  .. _sensors-examples:
+  ```
 
-    and point to it as from this, or from other documents as:
+  and point to it as from this, or from other documents as:
 
-           :ref:`sensors-examples`
-           :ref:`My examples <sensors-examples>`
+  ```rst
+  :ref:`sensors-examples`
+  :ref:`My examples <sensors-examples>`
+  ```
 
-    Note that the leading `_` underscore is gone, and the reference is quoted.
+  Note that the leading `_` underscore is gone, and the reference is quoted.
 
-    Name convention for references is `_filename-refname` (because they are unique across the docs).  Note that there is no way to reference just a point in the docs. See http://sphinx-doc.org/markup/inline.html#cross-referencing-syntax
+  Name convention for references is `_filename-refname` (because they are unique across the docs).  Note that there is no way to reference just a point in the docs. See http://sphinx-doc.org/markup/inline.html#cross-referencing-syntax
 
 * External links:
 
-    `External link <http://webchat.freenode.net/?channels=stackstorm>`_
+  ```rst
+  `External link <http://webchat.freenode.net/?channels=stackstorm>`_
+  ```
 
 * Inlcude a document, full body:
 
-    .. include:: /engage.rst
+  ```rst
```

Comment:
As we're fixing typos - can we fix the "Inlcude" typo in the line above?

Reply:
Done